narrow_valley.py

```diff
@@ -0,0 +1,647 @@
+import time
+import random
+
+#<------------------------------------------------------------ Basic Functions ------------------------------------------------------------>    
+
+def coin_flip():
+	return random.choice(['heads', 'tails'])
+
+def die_roll():
+	return random.randint(1, 6)
+
+def time_print(string):
+    print(string)
+    time.sleep(2)
+
+def time_print_loop(lst):
+    for i in range(len(lst)):
+        time_print(lst[i])
+
+def time_print_img(lst):
+    for i in range(len(lst)):
+        print(lst[i])
+        time.sleep(.5)
+
+def valid_input(promt, option1, option2):
+    while True:
+        response = input(promt).lower()
+        if option1 in response:
+            return option1
+        if option2 in response:
+            return option2
+        else:
+            time_print("I don't understand.")
+    return response
+
+def contine_on():
+    while True:
+        response = input("Enter (1) to continue.\n").lower()
+        if "1" in response:
+            break
+        else:
+            time_print("I don't understand.")
+
+#<------------------------------------------------------------ Fight System ------------------------------------------------------------>    
+
+def intro_fight():    
+    time_print('You are both still, waiting for the right time to make your move, and then...\n')
+
+
+def choose_stats(items):
+    opt1 = {
+        'boss_hp': 15, 
+        'player_hp': 15,
+        'boss_name': 'Elijah'
+        }
+    opt2 = {
+        'boss_hp': 15, 
+        'player_hp': 15, 
+        'boss_name': 'Clover',
+        }
+    if items['key'] == '*Primal Command*':
+        items.update(opt1)
+    else:
+        items.update(opt2)
+    return items
```

Comment:
Can you add some documentation that explains what you're doing you're doing here and why? It's not obvious what is going on from the code itself.